lug/config.yaml

```diff
@@ -14,10 +14,162 @@ dummy:
     retry_interval: 60
 
 repos:
-  - type: shell_script
+  #archlinux
+  - type: shell_script 
     script: /worker-script/rsync.sh
-    source: rsync://archive.ubuntu.com/ubuntu
+    source: rsync://rsync.archlinux.org/ftp_tier1/
```

Comment:
We do not have tier1 access to archlinux.org. You may run `rsync -avH <url> --dry-run --stats` on your local machine to see if the repo could be correctly accessed.

By the way, we already have a full clone of archlinux at mirrors.sjtug, so we could remove this entry.

Reply:
decide to remove this entry